Sources/SwiftParserDiagnostics/ParseDiagnosticsGenerator.swift

```diff
@@ -645,9 +645,48 @@ public class ParseDiagnosticsGenerator: SyntaxAnyVisitor {
     }
 
     if node.label.isMissing {
+      let replaceTokens: [TokenSyntax] = [node.unexpectedBetweenLabelAndColon].compactMap { $0?.tokens(viewMode: .sourceAccurate) }.flatMap { $0 }
+
+      let versionMessage: ParserFixIt =
+        node.unexpectedBetweenLabelAndColon == nil
+        ? InsertFixIt(tokenToBeInserted: .identifier("_version"))
+        : ReplaceTokensFixIt(replaceTokens: replaceTokens, replacements: [.identifier("_version"), .colonToken()])
+
+      let underlyingVersionMessage: ParserFixIt =
+        node.unexpectedBetweenLabelAndColon == nil
+        ? InsertFixIt(tokenToBeInserted: .identifier("_underlyingVersion"))
+        : ReplaceTokensFixIt(replaceTokens: replaceTokens, replacements: [.identifier("_underlyingVersion"), .colonToken()])
+
```

Comment:
It seems there's some problems with GitHub (maybe?) that my former commit didn't sync in the pr, so I re-pushed it again, which may lead the Compare button not showing anything.

Here are the major changes for your convenience.